guardian/recipes-backend
# Publish status messages (read in Fronts tool)

## What does this change?

Publishes status messages on curation deploy succeed or failure to an SNS topic, which is read in the Fronts tool.

## How to test

- The automated tests should pass.
- Running in CODE, attempt to publish a Feast issue in Fronts CODE. You should see confirmation of a publish in the event list.

Tested in CODE:

<img width="766" alt="Screenshot 2024-08-02 at 13 52 29" src="https://github.com/user-attachments/assets/3597934d-5fa8-4704-b198-af6554e1b04b">
